### ISSUE TITLE
MLS league_table is only pulling the Eastern Confrence

### ISSUE THREAD

**Issue by dougmellon** [JaseZiv/worldfootballR#71]
```
mls_league_table_2020 <- get_season_team_stats(country = "USA", 
                                               gender = "M", 
                                               season_end_year = "2020", 
                                               tier = "1st", 
                                               stat_type = "league_table")
```

When using the above, it only returns the table for the Eastern Conference.

**Comment by JaseZiv** [comment]
Thanks for bringing this to my attention.

I'll work on a solution to this. In your opinion which of the two options below do you prefer?

a. Add an extra column to a data frame returning both conferences
b. An extra argument to pass to the function to specify conference (knowing this will only be relevant for a few leagues)

**Comment by dougmellon** [comment]
I would say option A but I'll leave that up to the masses. I was going to take a jab at it but my limited knowledge in R will likely take too long.

**Comment by JaseZiv** [comment]
@dougmellon This should hopefully be fixed in `v0.4.6`. This version isn't yet on CRAN, so install from GitHub.

Let me know if there's any issues.

Jase
